### ISSUE TITLE
Profile Backup to 3box is Failing

### ISSUE THREAD

**Issue by walidmujahid** [gitcoinco/web#5960]
The new profile backup to 3box is not working and I am recieving "Failed to backup profile data to 3Box. Please try again." error. 

It would also seem that when attempting tot POST to `api/v0.1/profile/backup`, a 500 response is given. The recieved response is:

{"active": "error", "code": 500, "nav": "home", "title": "Error 500"}

The logs from the Console are:

```
raven.js:58 ############################################
raven.js:58 # Hi there technical person,
raven.js:58 # 
raven.js:58 # We'd love your help 
raven.js:58 # **** Pushing Open Source Forward **** 
raven.js:58 # Get in touch: 
raven.js:58 # - http://github.com/gitcoinco
raven.js:58 # - founders@gitcoin.co
raven.js:58 # Turnaround a Funded Issue: 
raven.js:58 # - https://gitcoin.co/explorer
raven.js:58 # 
raven.js:58 # <3 
raven.js:58 # Gitcoin Core
raven.js:58 ############################################

raven.js:58 check profile backup flag true
raven.js:58 start sync data to 3box

raven.js:1543 POST https://gitcoin.co/api/v0.1/profile/backup 500
(anonymous) @ raven.js:1543
fetchProfieData @ profile-backup.458ccb12f6b1.js:148
startProfileDataBackup @ profile-backup.458ccb12f6b1.js:167
(anonymous) @ profile-backup.458ccb12f6b1.js:232
d @ raven.js:445
setTimeout (async)
(anonymous) @ raven.js:1202
(anonymous) @ profile-backup.458ccb12f6b1.js:228
l @ jquery.a09e13ee94d5.js:2
c @ jquery.a09e13ee94d5.js:2
d @ raven.js:445
setTimeout (async)
(anonymous) @ raven.js:1202
(anonymous) @ jquery.a09e13ee94d5.js:2
u @ jquery.a09e13ee94d5.js:2
fireWith @ jquery.a09e13ee94d5.js:2
fire @ jquery.a09e13ee94d5.js:2
u @ jquery.a09e13ee94d5.js:2
fireWith @ jquery.a09e13ee94d5.js:2
ready @ jquery.a09e13ee94d5.js:2
_ @ jquery.a09e13ee94d5.js:2
raven.js:58 data null
raven.js:58 Swarm listening on /dns4/p2p.3box.io/tcp/9090/wss/p2p-websocket-star/ipfs/QmZcDBDfXuX6L34HSgJV2DA76XCWmS3uRq869fHKyzDm22
raven.js:58 Swarm listening on /p2p-circuit/ipfs/QmZcDBDfXuX6L34HSgJV2DA76XCWmS3uRq869fHKyzDm22
raven.js:58 Swarm listening on /p2p-circuit/dns4/p2p.3box.io/tcp/9090/wss/p2p-websocket-star/ipfs/QmZcDBDfXuX6L34HSgJV2DA76XCWmS3uRq869fHKyzDm22
raven.js:58 openBox succeeded
raven.js:58 sync complete
raven.js:58 sync done in space GitCoin
raven.js:58 backup data into space

raven.js:1543 POST https://gitcoin.co/api/v0.1/profile/backup 500
(anonymous) @ raven.js:1543
fetchProfieData @ profile-backup.458ccb12f6b1.js:148
backupProfile @ profile-backup.458ccb12f6b1.js:45
(anonymous) @ profile-backup.458ccb12f6b1.js:182
onSyncDone @ profile-backup.458ccb12f6b1.js:35
(anonymous) @ 3box.min.js:176
u @ 3box.min.js:80
(anonymous) @ 3box.min.js:80
E.forEach.e.<computed> @ 3box.min.js:80
r @ 3box.min.js:37
a @ 3box.min.js:37
Promise.then (async)
r @ 3box.min.js:37
a @ 3box.min.js:37
(anonymous) @ 3box.min.js:37
(anonymous) @ 3box.min.js:37
(anonymous) @ 3box.min.js:176
(anonymous) @ 3box.min.js:176
u @ 3box.min.js:80
(anonymous) @ 3box.min.js:80
E.forEach.e.<computed> @ 3box.min.js:80
r @ 3box.min.js:37
a @ 3box.min.js:37
Promise.then (async)
r @ 3box.min.js:37
a @ 3box.min.js:37
Promise.then (async)
r @ 3box.min.js:37
a @ 3box.min.js:37
(anonymous) @ 3box.min.js:37
(anonymous) @ 3box.min.js:37
(anonymous) @ 3box.min.js:176
(anonymous) @ 3box.min.js:80
u @ 3box.min.js:80
(anonymous) @ 3box.min.js:80
E.forEach.e.<computed> @ 3box.min.js:80
r @ 3box.min.js:37
a @ 3box.min.js:37
(anonymous) @ 3box.min.js:37
(anonymous) @ 3box.min.js:37
(anonymous) @ 3box.min.js:80
openSpace @ profile-backup.458ccb12f6b1.js:39
(anonymous) @ profile-backup.458ccb12f6b1.js:180
(anonymous) @ profile-backup.458ccb12f6b1.js:21
Promise.then (async)
(anonymous) @ profile-backup.458ccb12f6b1.js:16
Promise.then (async)
openBox @ profile-backup.458ccb12f6b1.js:15
startProfileDataBackup @ profile-backup.458ccb12f6b1.js:179
async function (async)
startProfileDataBackup @ profile-backup.458ccb12f6b1.js:167
(anonymous) @ profile-backup.458ccb12f6b1.js:232
d @ raven.js:445
setTimeout (async)
(anonymous) @ raven.js:1202
(anonymous) @ profile-backup.458ccb12f6b1.js:228
l @ jquery.a09e13ee94d5.js:2
c @ jquery.a09e13ee94d5.js:2
d @ raven.js:445
setTimeout (async)
(anonymous) @ raven.js:1202
(anonymous) @ jquery.a09e13ee94d5.js:2
u @ jquery.a09e13ee94d5.js:2
fireWith @ jquery.a09e13ee94d5.js:2
fire @ jquery.a09e13ee94d5.js:2
u @ jquery.a09e13ee94d5.js:2
fireWith @ jquery.a09e13ee94d5.js:2
ready @ jquery.a09e13ee94d5.js:2
_ @ jquery.a09e13ee94d5.js:2

```

It is possible this is causing a secondary problem for the active MetaMask account when login attempts are made for the 3box hub: https://github.com/3box/3box-dapp/issues/610 

I am using Chrome stable  80.0.3987.87 and MetaMask 7.7.4.

**Comment by owocki** [comment]
@thelostone-mc @octavioamu mind linking the fix PR here?

**Comment by oznekenzo** [comment]
@owocki 3box here, wondering what 3box.js version you guys are using, have tried to replicate the process of syncing with 3box, but seems the feature is flagged at the moment

**Comment by octavioamu** [comment]
@oznekenzo @think-in-universe already fixed here #5955
